ddpui/tests/core/test_llm_summarization.py

```
@@ -0,0 +1,265 @@
+            mock_get_job_info.return_value = {"job": {"configId": "test-connection"}}
+
+            try:
+                summarize_logs(
+                    orguser_id=None,
+                    type=LogsSummarizationType.AIRBYTE_SYNC,
+                    job_id=12345,
+                    connection_id="test-connection",
+                )
+            except Exception:
+                # We expect it to fail at some point, but we're testing the org extraction
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

(SIM105)